### PR TITLE
Have added support to simple_marker to allow script to be included in popups

### PR DESCRIPTION
**Pull request by samchorlton**
Have added functionality to branca to support html in popups and have amended map.py and folium.py so that this can be set as a flag.

### CODE REVIEW

**Comment by samchorlton** [comment]
Will ensure Pep 8 compliancy

File: folium/folium.py [hunk 1]
```diff
@@ -188,7 +189,7 @@ def simple_marker(self, location=None, popup=None,
             raise ValueError("%s is deprecated. Use %s instead" %
                              ("clustered_marker", "MarkerCluster"))
         if isinstance(popup, text_type) or isinstance(popup, binary_type):
-            popup_ = Popup(popup, max_width=popup_width)
+            popup_ = Popup(popup, script, max_width=popup_width)
```
Comment (BibMartin) [inline folium/folium.py]: It shall be `script=script` here.

File: folium/map.py [hunk 2]
```diff
@@ -72,13 +72,14 @@ class LegacyMap(MacroElement):
...
-        May be used for maps of flat surfaces (e.g. game maps). Note that the y axis
-        should still be inverted (going from bottom to top).
+        * EPSG4326 : A common CRS among GIS enthusiasts. Uses simple
+        Equirectangular projection.
+        * EPSG3395 : Rarely used by some commercial tile providers. Uses
+        Elliptical Mercator projection.
+        * Simple : A simple CRS that maps longitude and latitude into x and y
+        directly. May be used for maps of flat surfaces (e.g. game maps).
+        Note that the y axis should still be inverted
+        (going from bottom to top).
```
Comment (BibMartin) [inline folium/map.py]: You don't need to bother about that pep8 things ; we accept up to 100 chars per line.

File: folium/map.py [hunk 3]
```diff
@@ -519,7 +524,7 @@ def __init__(self, html=None, max_width=300):
         if isinstance(html, Element):
             self.html.add_children(html)
         elif isinstance(html, text_type) or isinstance(html, binary_type):
-            self.html.add_children(Html(text_type(html)))
+            self.html.add_children(Html(text_type(html), script))
```
Comment (BibMartin) [inline folium/map.py]: `script=script`

**Comment by samchorlton** [comment]
Have updated script tags as suggested

**Comment by samchorlton** [comment]
Any ideas why the tests are failing?

**Comment by BibMartin** [comment]
Yes, this is due to the fact that your other PR is not incorporated yet in `branca`.

When it's merged, you'll need to replace branca's commit number in `.travis.yml` so that the tests checkout your new version of branca.

**Comment by BibMartin** [comment]
Now that your banca PR is ok, you need to replace a27055b3a9bd823a917a9a9847efac8dbd590238 by 36e7284971b3476d97a7e330d4b6c64969703b85 in `.travis.yml`

**Comment by ocefpaf** [comment]
Based on your comments above we should start testing `folium` against a released and latest master of `branca`.  I can do the `.travis.yml` configuration.  @BibMartin can you make a branca `v0.1.0` release?

**Comment by ocefpaf** [comment]
@BibMartin this is how Travis-CI results will look like:

https://travis-ci.org/ocefpaf/folium

I still need to fix a few things here and there and then we can change the commit has to a release.

**Comment by BibMartin** [comment]
@ocefpaf
Most probably the branca_release part will always fail on master, because changes will be done to folium to adapt it to branca changes.
This PR is a good example for it.

I'll do a branca release as soon as I have plugged vital things (docs, examples).

**Comment by samchorlton** [comment]
Do you want me to wait to make any changes for now?

**Comment by ocefpaf** [comment]
> Most probably the branca_release part will always fail on master, because changes will be done to folium to adapt it to branca changes.

Branca changes will break the `latest master` test, but that is OK. The `latest master` test is just a tool to analyse how `folium` takes the changes. Passing the `released` test will be mandatory for merges though.

**Comment by BibMartin** [comment]
@samchorlton Now that #373 is merged, I think you can rebase your PR and this may work (hint: http://stackoverflow.com/questions/21148512/git-rebase-from-one-branch-to-another).

**Comment by ocefpaf** [comment]
@samchorlton any change for you to rebase this? Also, can you add a test and an entry in the changelog?

**Comment by ocefpaf** [comment]
I will rebase this in another PR. Thanks @samchorlton and sorry for the long time it took to act on this PR :flushed:

**Pull request closed without merging**